### PR TITLE
Add instructions for syntax highlighting on all .rc files

### PR DESCRIPTION
**Pull request by MetRonnie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,26 @@ Download this repository keeping the `.tmbundle` extension.
     ``git clone https://github.com/cylc/Cylc.tmbundle <path to sublime packages>/Cylc.tmbundle``
 
    (to get package path click Preferences => Browse Packages)
+   
+## FAQ - How do I enable syntax highlighting for all `.rc` files?
+
```
Comment (oliver-sanders) [inline README.md:22]: For context would be good to add something along the lines of:

By default this bundle will associate `suite.rc` and `.cylc` files with the Cylc language, however, if you are working with Cylc files which don't match these names...

**Review by oliver-sanders — COMMENTED** [review]
For some users this might be a good compromise, after all the cylc highlighter should do a reasonable job of `.rc` files anyway.

Would be good to add a few words explaining why users would want to add `.rc` files:

File: README.md (hunk 1 above)
Comment (MetRonnie) [reply README.md:22]: Added

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Cylc TextMate Bundle
 
-Last updated: 2020-06-26
+Last updated: <span actions:bind='update-date'>2020-06-26</span>
```
Comment (MetRonnie) [inline README.md:3]: Might as well put this in too. Should prevent anyone accidentally breaking the updating of this date

Comment (oliver-sanders) [reply README.md:3]: Meh, I think it was safe enough before but this is fine too.

(FYI `actions:bind` is an illegal attribute)

File: README.md (hunk 1 above)
Comment (oliver-sanders) [reply README.md:22]: Thanks.

File: README.md (hunk 2 above)
Comment (MetRonnie) [reply README.md:3]: Well, the browser strips `actions:bind` out, which is fine

Comment (oliver-sanders) [reply README.md:3]: If you prefix it with `data-` then it's valid HTML.

Or simpler still just use an `id`:

`<span id="update-date">2020-06-26</span>`

Comment (MetRonnie) [reply README.md:3]: Trouble with `id` is it isn't clear this is a sort of data binding used by Actions. Could use `data-actions-bind` but it's extra characters :P

**Pull request merged**